components/YamlEditor.vue

```diff
@@ -142,6 +142,14 @@ export default {
       if (neu) {
         this.$refs.yamluploader.click();
       }
+    },
+
+    value(neu) {
```

Comment:
Are you sure this is needed? We had issues with putting this in in the YamlEditor as it caused a loop of change events across the 2-way binding